src/components/search/_search.scss

```diff
@@ -102,10 +102,6 @@
   }
 
   .#{$prefix}--search--lg .#{$prefix}--search-close {
-    right: rem(55px);
```

Comment:
I remember @aledavila brought this back recently. That said, is this change intentional…?

Reply:
Yes, removing this as it was throwing off the positioning of the close icon. The close icon should always be `12px` from the right